Infrastructure/output.tf

```diff
@@ -9,5 +9,5 @@ output "private_key" {
 }
 
 output "volume_id" {
```

Comment:
Meaningful naming :)
Always think like you have hundreds of machines and not only one.
Later if you will want to refactor this Action for spinning up 50 machines what will "public_ip" say to the operator?
A good name will be "bastion_server_public_ip".

Reply:
done!